Makefile

```diff
@@ -219,7 +219,7 @@ endif
 
 # gvisor-tap-vsock version for gvproxy.exe and win-sshproxy.exe downloads
 # the upstream project ships pre-built binaries since version 0.7.1
-GV_VERSION=v0.8.4
+GV_VERSION=$(shell grep github.com/containers/gvisor-tap-vsock go.mod  | cut -d" " -f2)
```

Comment:
I think it would make sense to rename this to GVPROXY_VERSION for some better consistency

Reply:
Done